day5/README.md

```diff
@@ -0,0 +1,29 @@
+Question of the day: https://leetcode.com/problems/path-sum/#/description
+
+"Given a binary tree and a sum, determine if the tree
+has a root-to-leaf path such that adding up all the
+values along the path equals the given sum."
+
+## Ideas ##
+
+The tree isn't organized, i.e. it's not a binary search
+tree. That means I'll have to do some kind of extensive
+searching, and since I want a path, DFS should work.
```

Comment:
or generalize to any n-ary tree. or any graph. then start doing runtime and space analyses on each of these solutions.